fix_includes_test.py

```diff
@@ -335,6 +335,80 @@ class Alert;                         ///-
 The full include-list for empty_internal_namespace:
 namespace maps_transit_realtime { namespace service_alerts { class StaticServiceAlertStore; } }   // lines 5-5
 ---
+"""
+    self.RegisterFileContents({'empty_internal_namespace': infile})
+    self.ProcessAndTest(iwyu_output)
+
+  def testRemoveEmptyAllmanNamespace(self):
```

Comment:
I haven't quite understood what the "continued Allman" regex matches -- should there be a test with nested namespaces to cover that code? E.g.
```
namespace A { namespace B { namespace C
{
}
}
}
```

or however it looks.

Reply:
@kimgr the continued allman regex simply matches the { on the next line.
namespace A // allman start re
{ // allman continued re
} // namespace end re

It is only triggered when an allman namespace is found (namespace without any { on the same line.  I'll add that test case and see if I can handle it somehow but it is a mix of both styles so it might be a bit tricky.

Reply:
Sorry, just to clarify a bit more, the "allman" style is something I'm getting from clang-format.  We use allman style for our clang-format option which states to always break before braces so for nested namespaces, it would be like the test cases supplied here
namespace A
{
namespace B
{
namespace C
{
} // namespace C
} // namespace B
} // namespace A

In order to support this formatting style, I added the regex statements for allman style as well as a regex for finding the appropriate continuation { line.

Reply:
Logic and test cases were added for mixed namespaces.  Now all namespace styles should be supported (hopefully haha).